### ISSUE TITLE
Any option for single node?

### ISSUE THREAD

**Issue by tkang007** [ray-project/ray#7098]
Hello,
I am surprising this super simple and powerful framewok.
Is there any option for a single node only?  
Thanks.

**Comment by richardliaw** [comment]
Yep! `ray.init()`. Let me push a fix to the docs to make this clearer.

**Comment by tkang007** [comment]
Richardliaw,  Thanks for your quick response.  
When you update docs for single, please consider to add additional information for minimum usage on the develop environment, such as memory and object_store_memory of ray.init().

**Comment by richardliaw** [comment]
yep, https://github.com/ray-project/ray/pull/7099

Feel free to review.

**Comment by tkang007** [comment]
Hi,

What is the minimum/recommended value for memory, object_store_memory,driver_object_store_memory and redis_max_memory for developer's single node small virtual machine.  Or, what is the calculation logic the auto set value for single node.  
One place at docs for these information might be helpful.

**Comment by richardliaw** [comment]
Recommended values for memory are computer dependent.

More info can be found in https://ray.readthedocs.io/en/latest/memory-management.html

Feel free to reopen if you have more questions.
